src/ProgramBlockEditor.test.js

```diff
@@ -187,58 +191,60 @@ test('blocks', () => {
     // when selected Action is add, when you press any program blocks, an empty block (none command) will be added to the previous index and set selectedCommand to null
+    //getProgramBlocks(wrapper).at(0).getDOMNode().focus = mockFocusHandler;
     getProgramBlockAtPosition(wrapper, 0).simulate('click');
+    //expect(mockFocusHandler.mock.calls.length).toBe(1);
     expect(mockChangeHandler.mock.calls.length).toBe(1);
     expect(mockChangeHandler.mock.calls[0][0]).toStrictEqual(['none', 'forward', 'left', 'forward', 'left']);
     // No onSelectAction calls should have been made
     expect(mockSelectHandler.mock.calls.length).toBe(0);
 
     wrapper.setProps({program : mockChangeHandler.mock.calls[0][0]});
-    wrapper.setProps({selectedAction: {'commandName' : 'right', 'type': 'command'}});
-    // when selected Action is a command, change existing command at a clicked block to be selected command and set selected action back to null
-    getProgramBlockAtPosition(wrapper, 0).simulate('click');
-    expect(mockChangeHandler.mock.calls.length).toBe(2);
-    expect(mockChangeHandler.mock.calls[1][0]).toStrictEqual(['right', 'forward', 'left', 'forward', 'left']);
-    // No onSelectAction calls should have been made
-    expect(mockSelectHandler.mock.calls.length).toBe(0);
-
-    wrapper.setProps({program : mockChangeHandler.mock.calls[1][0]});
-    wrapper.setProps({selectedAction: {'action' : 'delete', 'type': 'editorAction'}});
-    // when selected Action is delete, when you press any program blocks, the block and its command will be removed from the program
-    getProgramBlockAtPosition(wrapper, 0).simulate('click');
-    expect(mockChangeHandler.mock.calls.length).toBe(3);
-    expect(mockChangeHandler.mock.calls[2][0]).toStrictEqual(['forward', 'left', 'forward', 'left']);
-    // No onSelectAction calls should have been made
-    expect(mockSelectHandler.mock.calls.length).toBe(0);
-
-    // repeat the test cases for the last command in the program
-
-    wrapper.setProps({program : mockChangeHandler.mock.calls[2][0]});
-    wrapper.setProps({selectedAction: {'action': 'add', 'type': 'editorAction'}});
-    // when selected Action is add, when you press any program blocks, an empty block (none command) will be added to the previous index and set selectedCommand to null
-    getProgramBlockAtPosition(wrapper, 3).simulate('click');
-    expect(mockChangeHandler.mock.calls.length).toBe(4);
-    expect(mockChangeHandler.mock.calls[3][0]).toStrictEqual(['forward', 'left', 'forward', 'none', 'left']);
-    // No onSelectAction calls should have been made
-    expect(mockSelectHandler.mock.calls.length).toBe(0);
-
-    wrapper.setProps({program : mockChangeHandler.mock.calls[3][0]});
-    wrapper.setProps({selectedAction: {'commandName' : 'right', 'type': 'command'}});
+    // wrapper.setProps({selectedAction: {'commandName' : 'right', 'type': 'command'}});
```

Comment:
There are a lot of commented out lines in this test. Please un-comment and fix as needed.